### DISCUSSION TITLE
Permalink Generation Discrepancy (Bug)

### DISCUSSION THREAD

**Discussion by Benjamin-SCAYLE** [GitbookIO/community#822] [3. Git Sync]
The permalink generation logic for group names containing an `&` has changed, causing inconsistencies between GitBook permalinks and GitLab folder names. Previously, the group name `Checkout & Frontend` generated a permalink of `checkout-and-frontend` (consistent with the GitLab folder name). After the latest update, GitBook now returns `checkout-frontend` as the permalink, which has broken our application. Gitlab still hat the foldername `checkout-and-frontend`

**Steps to Reproduce:**

1. Create a group with the name Checkout & Frontend in GitBook.
2. Check the generated permalink in GitBook (checkout-frontend).
3. Verify the folder name in GitLab (checkout-and-frontend).

**Expected Behavior:**
The permalink in GitBook should remain consistent with the GitLab folder name (`checkout-and-frontend`).

**Actual Behavior:** 
GitBook generates the permalink as `checkout-frontend`, while the GitLab folder remains `checkout-and-frontend`.

**Comment by utkuufuk** [comment]
Hi @Benjamin-SCAYLE,

Are you still experiencing this issue?

To clarify, you previously had the `checkout-and-frontend` folder on GitLab, which matched the permalink on GitBook. However, a few days ago, the GitBook permalink seemingly changed on its own to `checkout-frontend`. Is that correct?

In other words, did you follow the steps outlined in the "Steps to Reproduce" section above, or did the issue occur without any action on your part?
